### PR TITLE
Issue #97 Partial Completion

### PR DESCRIPTION
**Pull request by wcouturiaux**
##Overview
>This PR implements calling completedAction for sharing challenge and reloads the page on close of the invite component to load show challenger their badge on the progress page.

##Test Plan
>This was tested manually.

##Follow ups
>This needs to be followed up with testing to ensure only a challenger gets the share challenge badge.

### CODE REVIEW

File: src/components/Invite.js [hunk 1]
```diff
@@ -33,6 +34,11 @@ function Invite({ toggleInvite }) {
     navigator.clipboard.writeText(shareUrl);
   }
 
+  async function reloadPage() {
+    await delay(500);
```
Comment (rickyyx) [inline src/components/Invite.js]: I think we should remove this delay.

Comment (rickyyx) [reply src/components/Invite.js]: As an future optimization, we could actually optionally reload the page? For example, we only reload when we know for sure the badge has been awarded. We could probably have some local variables to keep track of the state. 
We could even add the badge locally w/o doing a full reload and fetching from the firestore.

**Comment by rickyyx** [comment]
As synced offline - @Devanshusp will help get this working.

File: src/components/Invite.js (hunk 1 above)
Comment (wcouturiaux) [reply src/components/Invite.js]: Delay is removed in updated PR. I agree in regards to tracking state or something. I had started looking into this to optimize how we reload or preferably only update the necessary piece(s)

**Comment by rickyyx** [comment]
Hey @wcouturiaux thanks for getting back at this! Could you also resolve the small conflict with the main development branch?

**Comment by wcouturiaux** [comment]
@githubxxcc can we get a review and get this merged? conflicts are resolved. A couple of small fixes got tagged on also - including one that partially addresses Issue#113

**Comment by rickyyx** [comment]
Thanks @wcouturiaux ! 

I created another follow-up issue on this: https://github.com/8by8-org/web-app/issues/115

Feel free to assign it to yourself if you have the time and are interested in working on this.

**Pull request merged**